eng/send-to-helix-step.yml

```diff
@@ -15,10 +15,16 @@ parameters:
   timeoutPerTestCollectionInMinutes: ''
   timeoutPerTestInMinutes: ''
   runCrossGen: ''
+  helixProjectFile: ''
 
 steps:
 - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
-  - powershell: eng\common\msbuild.ps1 -ci tests\helixpublishwitharcade.proj /maxcpucount
+  - powershell: eng\common\build.ps1 /p:DotNetPublishToBlobFeed=true -ci -restore -projects $(Build.SourcesDirectory)\eng\empty.proj
```

Comment:
It is unfortunate that this hack has to be added directly to the yml. I am currently using it in init-tools.sh. Can you add a comment to remove and consolidate this when we move to arcade. I want to avoid having this restore sprinkled explicitly over many places.

Reply:
Done.